src/components/Badge/__tests__/content.spec.js

```diff
@@ -1,27 +1,58 @@
+import React from 'react';
+import { mount } from 'enzyme';
 import Content from './../content';
 
 describe('<Content/>', () => {
     it('should return the label when there are not children', () => {
-        const label = 'Label in Content';
-        expect(Content({ label })).toBe(label);
+        const component = mount(
+            <Content label="Label in Content" />,
+        );
+        expect(component.find('span').text()).toBe('Label in Content');
     });
     it('should return the label when children is null', () => {
-        const label = 'Label in Content';
-        const children = null;
-        expect(Content({ label, children })).toBe(label);
+        const component = mount(
+            <Content label="Label in Content">
+                {null}
+            </Content>,
+        );
+        expect(component.find('span').text()).toBe('Label in Content');
     });
     it('should return the label when children is undefined', () => {
-        const label = 'Label in Content';
-        const children = undefined;
-        expect(Content({ label, children })).toBe(label);
+        const component = mount(
+            <Content label="Label in Content">
+                {undefined}
+            </Content>,
+        );
+        expect(component.find('span').text()).toBe('Label in Content');
```

Comment:
integrate this test with the above and use an array with all falsy values except 0
also add a test for the 0 case